spec/puppetserver/ca/local_certificate_authority_spec.rb

```diff
@@ -1,23 +1,50 @@
 require 'puppetserver/ca/local_certificate_authority'
+require 'puppetserver/ca/config/puppet'
 
+require 'utils/ssl'
 require 'puppetserver/ca/utils/signing_digest'
 
 RSpec.describe Puppetserver::Ca::LocalCertificateAuthority do
+  include Utils::SSL
 
   let(:settings) {
-    { :ca_ttl => 157680000,
-      :ca_name => 'bulla2',
-      :subject_alt_names => '',
-      :root_ca_name => 'bulla',
-      :certname => 'ulla',
-      :keylength => 512,
-      :hostprivkey => '$privatekeydir/$certname.pem',
-      :hostpubkey => '$publickeydir/$certname.pem',
-      :csr_attributes => '$confdir/csr_attributes.yaml',
-      :serial => '$cadir/serial' } }
+    with_ca_in(Dir.mktmpdir) do |config|
+      return Puppetserver::Ca::Config::Puppet.new(config).load({})
```

Comment:
The `with_ca_in` above yields a config file and the confdir. If we want to fully root this action in the tmpdir we need to override the confdir when we call load.

Reply:
I've added the confdir to the `load` method as needed.